### PR TITLE
feat(js/cli) add 'yes' flag

### PR DESCRIPTION
**Pull request by Zewasik**
Resolves # .

- fix path for clean build 
- refactor project generation
- make js example private

@osipov-mit

### CODE REVIEW

File: js/.prettierrc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (osipov-mit) [inline js/.prettierrc:1]: I don't think that this file is needed since we have eslint config

File: js/cli/src/generate/index.ts [hunk 2]
```diff
@@ -3,16 +3,141 @@ import { Sails } from 'sails-js';
...
+    this.isAutomaricOverride = isAutomaricOverride;
+
+    return this;
+  }
+
+  async build() {
+    const rootPath = this.projectPath[0];
+    const srcPath = path.join(...this.projectPath);
+
+    const libFilePath = this.isProject ? path.join(...this.projectPath) : this.projectPath[0];
```
Comment (osipov-mit) [inline js/cli/src/generate/index.ts:78]: If I got it right this path is not the path to lib.ts file but the path to the directory where lib.ts file is generated. If so, let's rename it to make it clearer

File: js/cli/src/generate/index.ts [hunk 3]
```diff
@@ -3,16 +3,141 @@ import { Sails } from 'sails-js';
...
+    }
+
+    if (!this.isProject) {
+      console.log(`Lib generated at ${libFilePath}`);
+      return;
+    }
+
+    if (!existsSync(srcPath)) {
+      mkdirSync(srcPath, { recursive: true });
+    }
```
Comment (osipov-mit) [inline js/cli/src/generate/index.ts:107]: Looks like it's created [here](https://github.com/gear-tech/sails/pull/562/files#diff-6475382e6a6d535733e40b183f13f7faa098c48741ae1df967cba39f26f875f4R80-R82)

File: js/.prettierrc (hunk 1 above)
Comment (Zewasik) [reply js/.prettierrc:1]: eslint does not provide formatting at the moment. I think it would be a good idea if everyone had the same formatting rules

**Review by osipov-mit — APPROVED** [review]

**Pull request merged**